src/helpers/index.js

```diff
@@ -1,5 +1,21 @@
 import { format } from 'date-fns';
 
+/**
+ * Format a UTC date
+ *
+ * @param {Date} startDate UTC date
```

Comment:
We're using a string and not a `date` object.

```suggestion
 * @param {string} date - UTC date
```